pyfr/plugins/fluidforce.py

```diff
@@ -39,6 +39,11 @@ def __init__(self, intg, cfgsect, suffix):
         # Boundary to integrate over
         bc = f'bcon_{suffix}_p{intg.rallocs.prank}'
 
+        # Moments origin
+        origin = self.cfg.get(cfgsect, 'morigin', '*')
```

Comment:
Since the only valid value is a literal hasopt and getliteral may be cleaner.  Also, compute the number of moments up front.

Reply:
Also cast to an np.array up front.